next/13/replace-next-head/index.ts

```diff
@@ -793,12 +824,105 @@ const collectTopLevelDefinitions = (
 		collectImportDeclaration(importDeclaration, topLevelVariablesContainer);
 	});
 };
+
+const isIdentifierDefinedInPageProps = (identifier: Identifier): boolean => {
+	const definitions = identifier.getDefinitionNodes();
+
+	return (
+		definitions.length !== 0 &&
```

Comment:
this is not needed in this context

Reply:
`[].every(() => false)` will return true; so if we check vars like process.env.somevar that have empty array as definitions, this code will break.